wenab-app/webpack.config.ts

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+// const webpack = require("webpack");
+const path = require("path");
```

Comment:
same question as in `generateClient.ts` - why JS lingo?

Reply:
Oh shit I guess 2 places 🤣